digdag-cli/src/main/java/io/digdag/cli/client/EnableSchedule.java

```diff
@@ -97,7 +101,9 @@ private void enableSchedule(Id scheduleId)
             throws IOException, SystemExitException
     {
         DigdagClient client = buildClient();
-        client.enableSchedule(scheduleId);
+        Optional<String> mode = modeString == null ? Optional.absent() : Optional.of(modeString);
```

Comment:
Just trivial things, but you may use Optional.fromNullable()

Reply:
Thank you, that's good to know. But I decided not to use this mode, so I'm going to mark as resolve this conversation.